backend/otel/otel_test.go

```diff
@@ -25,8 +49,52 @@ func TestHandler_HandleLog(t *testing.T) {
 	h.HandleLog(w, r)
 }
 func TestHandler_HandleTrace(t *testing.T) {
+	inputBytes, err := os.ReadFile("./samples/traces.json")
```

Comment:
Out of curiosity, how did you capture this data? Did you use something to sniff out network traffic?

Reply:
i intercepted some incoming otel traces in `otel.go` like so

![Screenshot 2023-04-14 at 1 14 46 PM](https://user-images.githubusercontent.com/1351531/232146251-0a80ba2f-cd5b-466a-8621-a422c24bd151.png)